plugins/flashrom/fu-flashrom-device.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ * Copyright (C) 2015-2017 Peter Jones <pjones@redhat.com>
+ *
```

Comment:
I think you'll want to update this to you rather than the file you used as a template.